cpp/lib/ome/xerces/dom/NamedNodeMap.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * #%L
+ * OME-XERCES C++ library for working with Xerces C++.
+ * %%
+ * Copyright © 2006 - 2014 Open Microscopy Environment:
```

Comment:
Copyright update is now completely handled via an automatic script/CI job so reviewing these in Bio-Formats should not be necessary.